README.md

```diff
@@ -255,6 +255,25 @@ Ekka implements a simple distributed lock service in 0.3 release.  The Lock APIs
 
 TODO:
 
+## Cluster without epmd
+
+The ekka 0.6.0 release implements erlang distribiton without epmd.
+
+See: http://erlang.org/pipermail/erlang-questions/2015-December/087013.html
+
+For example:
+
+```
+## Dist port: 4370
+erl -pa ebin -pa deps/*/ebin -proto_dist ekka -start_epmd false -epmd_module ekka_epmd -name node1@127.0.0.1 -s ekka
```

Comment:
Isn't `-proto_dist ekka_dist` ?

Reply:
Neither `ekka` or `ekka_epmd`, it should be `inet_tcp` or `inet_tls` or `inet6_tcp`

Reply:
No, it should be ekka_dist